apis/logging/v1/clusterlogging_types.go

```diff
@@ -364,25 +381,6 @@ type FluentdBufferSpec struct {
 	RetryTimeout FluentdTimeUnit `json:"retryTimeout"`
 }
 
```

Comment:
We are losing some of these annotations, is that an issue?

Reply:
I don't follow, what annotations do you mean. Maybe it was removed early. @jcantrill

Reply:
These annotations are mandatory:
1. The deep-copy-gen annotation controls the deep-copy method generation for our CRD types.
2. The open-api-gen annotation controls the generation of open-api validation spec in our CRD yaml files.
3. The subresource status is mandatory, as to provide a secure subresource for our CR to make updates on the status.
4. Categories is needed to do: `oc get cl` or `oc get logging` and you get all CRs tag by these
5. PrintColumn is a custom colum added for the ClusterLogging CR, when `oc get clusterlogging instance`

Reply:
@vparfonov based on @periklis comments here, please revert any annotations on the APIs that were deleted

Reply:
ok

Reply:
FYI: deep-copy annotation was removed it EO during upgrade SDK
https://github.com/openshift/elasticsearch-operator/pull/566/files#diff-1935a3b218b074c5e652ded36f38a7aa6c3fa08f1f8504d0698ad1edb49cc3ccL51